packages/material-ui-lab/src/TreeView/TreeView.js

```diff
@@ -5,7 +5,7 @@ import TreeViewContext from './TreeViewContext';
 import { withStyles } from '@material-ui/core/styles';
 
 export const styles = {
-  /* Styles applied to the root component. */
+  /* Styles applied to the root element. */
```

Comment:
Wrong PR?

Reply:
Thanks, yeah, rebasing, I have already applied it to your tree view iteration, it's on master.